velox/benchmarks/tpch/TpchBenchmark.cpp

```diff
@@ -57,7 +57,11 @@ DEFINE_string(data_path, "", "Root path of TPC-H data");
 DEFINE_int32(
     run_query_verbose,
     -1,
-    "Run a given query and print execution statistics ");
+    "Run a given query and print execution statistics");
+DEFINE_bool(
+    include_custom_stats,
+    false,
+    "Include custom statistics along with execution statistics");
```

Comment:
I think the description here is clear that this flag works only with `run_query_verbose` that prints the execution statistics.